src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/ECDsa/ECDsaImportExport.cs

```diff
@@ -0,0 +1,347 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//#define DEBUG_IMPORT_EXPORT
```

Comment:
> but keep the corresponding ifdefs

Why?

If you think OutputCurveForDebugging is still useful, a better pattern seems to me to be to remove all of the \#ifs and put \[Conditional("DEBUG_IMPORT_EXPORT")\] on the OutputCurveForDebugging method.

Same effect, but with a lot fewer \#ifs floating around.